community/kernel/src/test/java/org/neo4j/kernel/GraphDatabaseFacadeFactoryTest.java

```diff
@@ -143,7 +146,8 @@ public LifeSupport createLife()
             @Override
             protected EditionModule createEdition( PlatformModule platformModule )
             {
-                return null;
+                EditionModule edition = Mockito.mock( EditionModule.class, Answers.RETURNS_DEEP_STUBS.get() );
```

Comment:
`Mockito` is recommended public facade class that should decrease number of mockito classes that user code interact with